components/resources/library/src/commonTest/resources/test_setup.js

```diff
@@ -0,0 +1,2 @@
+// Setting this will make the tests use a blocking XMLHttpRequest instead of the asynchronous 'fetch'
+window.composeResourcesTesting = true;
```

Comment:
is it possible to put it into the webTest source set?

Reply:
fixed